lib/Parse/ParseDecl.cpp

```diff
@@ -631,9 +631,6 @@ bool Parser::parseSpecializeAttributeArguments(
       if (ParamLabel == "exported") {
         auto trueLoc = Tok.getLoc();
         bool isTrue = consumeIf(tok::kw_true);
-        if (isTrue) {
-          diagnose(trueLoc, diag::attr_specialize_export_true_no_op);
```

Comment:
I think you can remove this diagnostic now

Reply:
I did.